tests/integration/test_nutritional_d7.py

```
@@ -0,0 +1,467 @@
+from security.role import Role
+from tests.conftest import get_access, make_headers, session, session_commit
+
+_HOSPITAL = 1
+_ADM = 900001
+_ADM_NOT_FOUND = 999998
+
+_POST_ENDPOINT = f"/nutritional/patients/{_ADM}/d7"
+_GET_ENDPOINT = f"/nutritional/patients/{_ADM}/d7"
+
+
+def _put_endpoint(id):
```

Ruff (0.15.12)

[error] Function argument `id` is shadowing a Python builtin

(A002)